### ISSUE TITLE
Youtube video thumbnail disappears on scroll (black background)

### ISSUE THREAD

**Issue by arnisjuraga** [alvarotrigo/fullPage.js#2068]
### Description

Embed Youtube video in page.
Scroll to page - video looks good.
Scroll away from it - Youtube video switches to "black background" - controls stays, but Thumbnails disappears.

If You disable autoscroll, then video looks ok.

As on mobile device You cannot initiate PLAY without touch gestures, it's kind a important.
Is it Youtube or Fullpage issue?
### Link to isolated reproduction with no external CSS / JS

Example based on #1575 jsfiddle
http://jsfiddle.net/q884Lv0a/
### Version

Tested on:
Chrome 50.0.2661.102
Firefox  46.0.1

**Comment by alvarotrigo** [comment]
That's an interesting one. I'll try to see what I can do about it. 
Thanks for reporting it!

**Comment by arnisjuraga** [comment]
its kind a because Fullpage.js tries to "play it".
If I setup `data-keepplaying`, then Fullpage for real - does not "start" playbeck, and then it's ok.

You can see in Youtube iframe, that it changes thumbnail clas to hidden, but video does not has started yet.

**Comment by alvarotrigo** [comment]
> its kind a because Fullpage.js tries to "play it".

Nop, quite the opposite. It tries to pause it :)

**Comment by alvarotrigo** [comment]
[Reproduction of the issue without fullpage.js](https://jsfiddle.net/h9o1g2c5/3/)

I've reported it in Youtube issues. No answer yet.

**Comment by aunruh** [comment]
Hey alvarotrigo

https://developers.google.com/youtube/iframe_api_reference

If you pause youtube videos this way the preview image doesnt disappear: 
var player = new YT.Player("#selector");
player.stopVideo();

I'm not super sure right now if that makes the video go back to time 0 instead of pause.

**Comment by arnisjuraga** [comment]
Isn't it fixed on Youtube side already? 
Both examples now looks correctly both in Firefox and Chrome.

**Comment by alvarotrigo** [comment]
@aunruh nop, it seems to be showing a black image when paused before starting it. Reproduction: https://jsfiddle.net/h9o1g2c5/5/

> Isn't it fixed on Youtube side already? 
> Both examples now looks correctly both in Firefox and Chrome.

What do you mean? We are discussing that when pausing a video which was not playing its preview will disappear.

**Comment by arnisjuraga** [comment]
Sorry, my bad. You are right, 
> when pausing a video which was not playing

**Comment by lorenzomarri** [comment]
Hi guys, I have the same issue with the black background on embed youtube videos.
I was wondering if you found a solution for it?

Thanks!

**Comment by alvarotrigo** [comment]
> I was wondering if you found a solution for it?

The topic still open. Which means no solution was found.

**Comment by jotajotaplapo** [comment]
Hi, i have the same issue in www.surveyteam.cl

**Comment by alvarotrigo** [comment]
@johanhermansson that's a [known and unresolved youtube bug](https://code.google.com/p/gdata-issues/issues/detail?id=7584&q=label%3AAPi-YouTube%20black&colspec=API%20ID%20Type%20Status%20Priority%20Stars%20Summary). Not a fullPage.js one.

**Comment by alvarotrigo** [comment]
I'll close it as there's nothing to do until Youtube fixes the problem.

**Comment by RaidoDev** [comment]
Seems to be fixed. https://issuetracker.google.com/issues/35176958
```Status: Won't Fix (Not Reproducible)```

**Comment by alvarotrigo** [comment]
@RaidoDev awesome! Thanks for letting me know!
I'll close the issue.
